pyro/distributions/torch_distribution.py

```diff
@@ -280,6 +285,108 @@ def variance(self):
         return self.base_dist.variance
 
 
+class ReshapedDistribution(TorchDistribution):
```

Comment:
Can we name this `ExpandedDistribution`? The original `Reshaped` implemented both `Independent()` and `.expand()`, whereas your new implementation serves only to implement `.expand()`.

Reply:
Good point, will change.